docs/config.rst

```diff
@@ -607,6 +607,7 @@ For example::
         }
     }
 
+.. tip:: To use an AWS China region, please specify the appropriate region code in ``endpoint``. For example ``aws.client.endpoint = "s3.cn-north-1.amazonaws.com.cn"``
```

Comment:
Does not the endpoint start with `https://`?

Reply:
On the comment which confirmed this setting, there's no `https://` - but yeah, I agree we should be more explicit on our end.

Reply:
```suggestion
.. tip:: To use an AWS China region, please specify the appropriate region code in ``endpoint``. For example ``aws.client.endpoint = "https://s3.cn-north-1.amazonaws.com.cn"``
```

Reply:
Now that you are mentioning it, I'm not sure anymore. Actually, in the docs there are no protocol prefix. 

https://docs.aws.amazon.com/general/latest/gr/s3.html#s3_website_region_endpoints

This we should to a test to verify it

Reply:
Regarding this, I was able to test access with `https`. Also, this is supported by 


- The [virtual hosting](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) docs.

- The s3 endpoint docs 


![2021-11-10 12 47 13](https://user-images.githubusercontent.com/12799326/141099687-35da6540-1902-42c7-b9b3-4d23f7ca1346.gif)